p

```diff
@@ -0,0 +1,87 @@
+diff --git a/integration/test_general_cli.py b/integration/test_general_cli.py
```

Comment:
What is this whole file?

Reply:
hmm.. I have no idea what that is and how that sneaked in. @meejah do you know?

Reply:
oh, weird that's from another PR of mine I think